packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -599,8 +599,7 @@ export class MgtPerson extends MgtTemplatedComponent {
         @click=${this.handleMouseClick}
         @mouseenter=${this.handleMouseEnter}
         @mouseleave=${this.handleMouseLeave}
-        @keydown=${this.handleKeyDown}
-        tabindex=0>
```

Comment:
Is this intentional? it feels like this should be a tab stop if there is either a click or hover behavior for this person

Reply:
Yes. In the people-picker, this causes every person in the dropdown to be focused on when you tab. I have removed it here so that if you need to focus on a person component when you tab, you can set it where you use it.